src/Modules/Visualization/ShowField.cc

```diff
@@ -430,6 +430,96 @@ void GeometryBuilder::renderFaces(
 
 
 
+inline static void writeAtributeToVBO(const Point& point, spire::VarBuffer* vboBuffer)
+{
+  vboBuffer->write(static_cast<float>(point.x()));
```

Comment:
These could be simplified with a template as well:
```
template <typename T>
void writeFloats(spire::VarBuffer* vboBuffer, std::initializer_list<T> ts)
{
  for (const T& t : ts)
   vboBuffer->write(static_cast<float>(t);
}
```
then each of these can pass a list, avoiding duplication:
```
writeFloats(vboBuffer, {point.x(), point.y(), point.z()});
```